spring-cloud-modules/spring-cloud-gateway/pom.xml

```diff
@@ -65,6 +65,14 @@
             <artifactId>jakarta.validation-api</artifactId>
             <version>${jakarta.validation-api.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-cache</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+        </dependency>
```

Comment:
Is this used - caffeine?

Reply:
Yes, refer to https://github.com/spring-cloud/spring-cloud-gateway/issues/2819

Reply:
Ok thanks